e2e-tests/kubernetes-headlamp-ci.yaml

```diff
@@ -10,34 +23,66 @@ spec:
     - port: 80
       targetPort: 4466
   selector:
-    k8s-app: headlamp
+    app.kubernetes.io/name: headlamp
+    app.kubernetes.io/instance: headlamp
 ---
-kind: Deployment
+# Source: headlamp/templates/deployment.yaml
 apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: headlamp
   namespace: kube-system
+  labels:
+    app.kubernetes.io/name: headlamp
+    app.kubernetes.io/instance: headlamp
 spec:
   replicas: 1
   selector:
     matchLabels:
-      k8s-app: headlamp
+      app.kubernetes.io/name: headlamp
+      app.kubernetes.io/instance: headlamp
   template:
     metadata:
       labels:
-        k8s-app: headlamp
+        app.kubernetes.io/name: headlamp
+        app.kubernetes.io/instance: headlamp
     spec:
+      initContainers:
+      - command:
+        - /bin/sh
+        - -c
+        - mkdir -p /build/plugins && cp -r /plugins/* /build/plugins/
+        image: ghcr.io/yolossn/headlamp-plugins:latest
```

Comment:
I guess this needs to change?

Reply:
What is different in that image than the default headlamp image? Why can't it be used?
(ghcr.io/headlamp-k8s/headlamp:latest)

Reply:
cc @joaquimrocha can we push the same at `headlamp-k8s` repository?

Reply:
Sure, done. Please review